### PR TITLE
Bookstore style

### PR DESCRIPTION
**Pull request by Mucyosoda**
This project will lay foundations for the Bookstore website.
React and Redux app was built and set up routing using React Router.
This app is able to add and delete books from the booklist by clicking on the add new book or delete button. The book can be stored on API and also delete from API
App styled according to recommended style in Zeplin
the project built-in :

React
Redux
webpack
babelrc
linter
API

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,253 @@
...
+  font-size: 1rem;
+}
+
+.removebtn:hover {
+  color: rgb(55, 55, 243);
+  border: 1px solid rgb(83, 83, 211);
+}
+
+.firstpart {
+  display: flex;
```
Comment (ZeenatLawal) [inline src/index.css]: - [ ] There are multiple declarations of display: flex, to follow best practices of CSS - DRY(Don't Repeat Yourself) I suggest creating a class for display flex to use where needed. Check this [article](https://speckyboy.com/good-bad-css-practices/) for more explanation.

Comment (ZeenatLawal) [reply src/index.css]: - [ ] Kindly check for other repetitions and modify them.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Mucyosoda, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -1,5 +1,252 @@
...
   margin: 0;
   padding: 0;
 }
+
+body {
+  margin: 20px;
+}
+
+.navigation-bar {
+  display: flex;
```
Comment (Lameck1) [inline src/index.css:14]: - [ ] Still on the concerns of the previous review. Not a new issue.
  
  Let's strive to keep the code DRY. This property:
  ```CSS
  display: flex;
  ```
  has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.
  
  ### Example code
  ```CSS
  .d-flex {
    display: flex;
    align-items: center;
  }
  ```
  
  Then apply this class to HTML element that needs this behavior:
  
  ```HTML
  <section class="d-flex">
  ```
  
  OR, in JSX, if creating elements dynamically:
  ```JSX
  <section className="d-flex">
  ```
  
  The goal of DRY is that we write this rule only once in the CSS. Please refactored accordingly just as was required in the first review.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Mucyosoda, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @Mucyosoda 

Woo ow 👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**